components/Card/Card.module.css

```diff
@@ -1,13 +1,15 @@
-.card {
+/* .card {
```

Comment:
@AlexVCS maybe we could remove this commented out code, since we are using Tailwind?

Reply:
good point @jonathans199! we can remove the whole file now